config/manager/kustomization.yaml

```diff
@@ -12,5 +12,5 @@ apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 images:
 - name: controller
-  newName: quay.io/red-hat-storage/mcg-osd-deployer
-  newTag: latest
+  newName: quay.io/osd-addons/mcg-osd
+  newTag: mcg-osd-deployer
```

Comment:
yes, Tag should be something that points to the version of the image, we keep it `mcg-osd-deployer` only for now and once we have prod ready version we should change the version to a valid one (eg:- 1.0.0)

Reply:
Okay, in that case, since we have a non-prod-ready alpha version on our hands, can we point this to say, `0.1.0` (or any other version that we can agree upon based on the number of changes and features hat have went in) to indicate that?

Reply:
Sure we can do that, will update the image tag